### PR TITLE
feat/messagebus

### PR DESCRIPTION
**Pull request by sulthonzh**
@nurcahyopujo

### CODE REVIEW

File: messagebus/producer.py [hunk 1]
```diff
@@ -0,0 +1,336 @@
...
+        )
+        sys.exit(1)
+
+    broker = sys.argv[1]
+    schema_registry_url = sys.argv[2]
+    username = sys.argv[3]
+    password = sys.argv[4]
+
+    # conf init
+    test_conf = {
```
Comment (nurcahyopujo) [inline messagebus/producer.py]: @goFrendiAsgard @sulthonzh what is python best practice for test code? 
should it be placed on different files?

**Comment by nurcahyopujo** [comment]
@goFrendiAsgard @sulthonzh what about static typing you guys discussed last time?

File: messagebus/producer.py (hunk 1 above)
Comment (goFrendiAsgard) [reply messagebus/producer.py]: I think so, this one should be located on `producer_test.py` or something. WDYT @sulthonzh?
I will recommend using pytest for this

Comment (sulthonzh) [reply messagebus/producer.py]: https://github.com/kata-ai/messagebus-kafka-python/pull/1/commits/c1d294dad74f47b4beea2ad0a3b1c044c5e6419e

**Pull request merged**